### DISCUSSION TITLE
problemas a correr no actions

### DISCUSSION THREAD

**Discussion by punkstarbr** [LITUATUI/M3UPT#39] [Q&A] [answered]
Olá, tudo bem? Fiz agora mesmo um fork no repositório, mas infelizmente não estou a conseguir fazer correr o push para atualizar a lista .m3u. Poderia dar-me alguma ajuda sobre o que posso fazer para conseguir atualizá-la? Estão a dizer que não tenho permissões necessárias para fazê-lo e "403" indica que o servidor rejeitou a solicitação por falta de autorização. Abraços.

**Comment by LITUATUI** [comment]
Olá.

O problema ainda persiste?

**Reply by punkstarbr** [reply]
Sim, ainda estão a falhar as atualizações executadas pelo Github Actions no fork do teu repositório. Sabes o que pode estar a acontecer? Além disso, tenho que fazer alguma configuração adicional para conseguir fazer push no meu próprio repositório? Abraços.

**Reply by LITUATUI** [reply]
Não sei onde está o problema, mas tenta ver nos outros forks que continuam activos, pode ser que descubras alguma diferença do teu que esteja a causar isso.

**Comment by LITUATUI** [comment, marked as answer]
Vai aos settings do teu repo e dá permissão de escrita em Actions - General.

![workflow-permissions](https://user-images.githubusercontent.com/49299328/230931973-c175476e-ce0e-42c0-a8ba-3e8542d5e6a4.png)

**Reply by punkstarbr** [reply]
Está a correr perfeitamente agora, obrigado, já estava a desistir, procurei a solução em muitos lugares
